python/TestHarness/TestHarness.py

```diff
@@ -581,6 +588,11 @@ def appendRecoverableTests(self, testers):
         testers.extend(new_tests)
         return testers
 
+    def addLibtorchGPUCLIParam(self, testers):
```

Comment:
Don't do this here. Just add it to `RunApp.getCommand()`

Reply:
Don't you need to remove this then?

Reply:
removed

Reply:
noice